data/tokens.json

```diff
@@ -3491,6 +3491,10 @@
       "gtmds_top-bar_color": {
         "value": "{Text.gtmds_color_text-color}",
         "type": "color"
+      },
+      "gtmds_top-bar_border-color": {
+        "value": "{gtmds_color_gray-3}",
+        "type": "color"
```

Comment:
Where is this used? I cannot see it applied on the top-bar, I think we should align the component with the styles from the new navi, WDYT?

Reply:
It used to be applied but figma branching might have messed it up. Let's keep it here and once everything is on main in Figma we can reapply it to avoid further conflicts like that